### PR TITLE
some cleanup + docs

### PR DESCRIPTION
**Pull request by rjaccard**
- Remove some codesmells (mostly TODO)
- Update the doc with the work of this semester
- some cleanup

Easier to review per commit but should also be fine by looking at all changes directly. Click on "view file" to see the schema I made.

### CODE REVIEW

File: be1-go/channel/lao/mod_test.go [hunk 1]
```diff
@@ -566,8 +566,8 @@ func (h *fakeHub) NotifyNewChannel(channeID string, channel channel.Channel, soc
 	h.Unlock()
 }
 
-// GetPubKeyOrg implements channel.HubFunctionalities
-func (h *fakeHub) GetPubKeyOrg() kyber.Point {
+// GetPubKeyOwner implements channel.HubFunctionalities
+func (h *fakeHub) GetPubKeyOwner() kyber.Point {
 	return h.pubKeyOrg
```
Comment (Anghul) [inline be1-go/channel/lao/mod_test.go:571]: Maybe change the name of h.pubKeyOrg

File: be1-go/channel/reaction/mod_test.go [hunk 2]
```diff
@@ -492,8 +492,8 @@ func generateKeys() (kyber.Point, kyber.Scalar) {
 	return point, secret
 }
 
-// GetPubKeyOrg implements channel.HubFunctionalities
-func (h *fakeHub) GetPubKeyOrg() kyber.Point {
+// GetPubKeyOwner implements channel.HubFunctionalities
+func (h *fakeHub) GetPubKeyOwner() kyber.Point {
 	return h.pubKeyOrg
```
Comment (Anghul) [inline be1-go/channel/reaction/mod_test.go:497]: Maybe change the name of h.pubKeyOrg

**Review by Anghul — APPROVED** [review]
Thank you for this work, juste some small comments (always the same)

File: be1-go/channel/chirp/mod_test.go [hunk 3]
```diff
@@ -461,8 +461,8 @@ func generateKeys() (kyber.Point, kyber.Scalar) {
 	return point, secret
 }
 
-// GetPubKeyOrg implements channel.HubFunctionalities
-func (h *fakeHub) GetPubKeyOrg() kyber.Point {
+// GetPubKeyOwner implements channel.HubFunctionalities
+func (h *fakeHub) GetPubKeyOwner() kyber.Point {
 	return h.pubKeyOrg
```
[the remark this answers is not in the document]
Comment (rjaccard) [reply be1-go/channel/chirp/mod_test.go]: Yes, sure! Nice catch 👍

File: be1-go/channel/consensus/mod_test.go [hunk 4]
```diff
@@ -1090,8 +1090,8 @@ func (h *fakeHub) RegisterNewChannel(channeID string, channel channel.Channel) {
 	h.Unlock()
 }
 
-// GetPubKeyOrg implements channel.HubFunctionalities
-func (h *fakeHub) GetPubKeyOrg() kyber.Point {
+// GetPubKeyOwner implements channel.HubFunctionalities
+func (h *fakeHub) GetPubKeyOwner() kyber.Point {
 	return h.pubKeyOrg
```
[the remark this answers is not in the document]
Comment (rjaccard) [reply be1-go/channel/consensus/mod_test.go]: 👍

File: be1-go/channel/election/mod_test.go [hunk 5]
```diff
@@ -464,8 +464,8 @@ func (h *fakeHub) NotifyNewChannel(channeID string, channel channel.Channel, soc
 	h.Unlock()
 }
 
-// GetPubKeyOrg implements channel.HubFunctionalities
-func (h *fakeHub) GetPubKeyOrg() kyber.Point {
+// GetPubKeyOwner implements channel.HubFunctionalities
+func (h *fakeHub) GetPubKeyOwner() kyber.Point {
 	return h.pubKeyOrg
```
[the remark this answers is not in the document]
Comment (rjaccard) [reply be1-go/channel/election/mod_test.go]: 👍

File: be1-go/channel/generalChirping/mod_test.go [hunk 6]
```diff
@@ -250,8 +250,8 @@ func (h *fakeHub) RegisterNewChannel(channeID string, channel channel.Channel) {
 	h.Unlock()
 }
 
-// GetPubKeyOrg implements channel.HubFunctionalities
-func (h *fakeHub) GetPubKeyOrg() kyber.Point {
+// GetPubKeyOwner implements channel.HubFunctionalities
+func (h *fakeHub) GetPubKeyOwner() kyber.Point {
 	return h.pubKeyOrg
```
[the remark this answers is not in the document]
Comment (rjaccard) [reply be1-go/channel/generalChirping/mod_test.go]: 👍

File: be1-go/channel/lao/mod_test.go (hunk 1 above)
Comment (rjaccard) [reply be1-go/channel/lao/mod_test.go]: 👍

File: be1-go/channel/reaction/mod_test.go (hunk 2 above)
Comment (rjaccard) [reply be1-go/channel/reaction/mod_test.go]: 👍

**Review by Anghul — APPROVED** [review]
LGTM

**Review by pierluca — APPROVED** [review]
LGTM

**Pull request merged**